File: example/__case_.py
import hmac

# 签名生成(sha256)
def newSignature(key, data):
    h = hmac.new(key.encode("utf-8"), data.encode("utf-8"), "sha256")
    return h.hexdigest()

if __name__ == "__main__":
    sign = newSignature("key", "123")
    print(sign)
    # a7f7739b1dc5b4e922b1226c9fcbdc83498dee375382caee08fd52a13eb7cfe2

File: example/__case_.php
<?php

// 签名生成(sha256)
function newSignature($key, $data) {
    return hash_hmac('sha256', $data, $key);
}

$sign = newSignature("key", "123");
echo $sign;
// a7f7739b1dc5b4e922b1226c9fcbdc83498dee375382caee08fd52a13eb7cfe2
